R/ParameterList.R

```diff
@@ -145,11 +146,17 @@ names.ParameterList <- function(x) {
 
 #' @describeIn ParameterList-Getter-Methods The parameter-list's parameter initial values
 #' @export
-initialValues.ParameterList <- function(object) {
-    vals <- lapply(object@parameters, initialValues)
-    name <- vapply(object@parameters, names, character(1))
-    names(vals) <- name
-    return(vals)
+initialValues.ParameterList <- function(object, nchains, ...) {
+    x <- lapply(
```

Comment:
can remove `x` assignment and directly have the `lapply()` as last value in this method